americangut/per_sample.py

```diff
@@ -515,3 +577,132 @@ def stage_per_sample_specific_statics(opts, sample_ids):
             result[id_] = "Cannot symlink for statics."
 
     return result
+
+
+def _plot_alpha(sample, alpha_map, alpha_field, group_field='SIMPLE_BODY_SITE',
+                output_dir=None, xlabel=None, fp=None, debug=False):
+    """Generates a distrbution plot for the data
+
+    Parameters
+    ----------
+    sample : str
+        The sample ID to be plotted
+    alpha_map_fp : str, dataframe
+        The filepath of a comma-seperated file or the pandas dataframe where
+        the sample ID are given in the `'#SampleID'` column, a column with
+        the name given by `alpha_field` contains alpha diversity values,
+        and the `group_field` column specifying the groups which should be
+        used to seperate the data for making the distribution plot.
+    alpha_field : str
+        The name of the column in `alpha_map` which includes the alpha
+        diversity values.
+    group_field : str
+        Default is 'SIMPLE_BODY_SITE'. The name of the column in `alpha_map`
+        which provides the grouping for generating distribution plots.
+    output_dir : str
+        The location where the alpha diversity figures should be saved.
+    xlabel : str
+        Text describing the quantity on the x-axis.
+
+    Returns
+    -------
```

Comment:
Because having multiple levels of returns means not having a tuple of Nones. This simplifies the input/output and targets whats needed.